config/canonical.json

```diff
@@ -31001,17 +31001,27 @@
   },
   "shop/supermarket|Мария-Ра": {
     "count": 177,
+    "countryCodes": ["ru"],
     "tags": {
       "brand": "Мария-Ра",
+      "brand:en": "Maria-ra",
+      "brand:wikidata": "Q4281631",
+      "brand:wikipedia": "ru:Мария-Ра",
       "name": "Мария-Ра",
+      "name:en": "Maria-ra",
       "shop": "supermarket"
     }
   },
   "shop/supermarket|Монетка": {
     "count": 401,
+    "countryCodes": ["ru"],
```

Comment:
According to [Overpass](https://overpass-turbo.eu/s/D1R) this is also not restricted to Russia. I don't know if this is fully true, as Монетка seems to translate to "Coin" so it could be a generic name in other countries?

Reply:
Official site of the network writes that they have stores only in Russia. For other countries, it may be a generic name

Reply:
I'm gonna accept that explanation 👍